students/JasneetChandok/Lesson04/mailroom2_v2.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3.6
+
+'''Lesson 3 - Mailroom Part 2'''
+
+# Importing Systems Module
+import sys
+
+
+# Starter list of existing donors
+donors = {'jasneet paramjit': [50, 200, 500],
+          'simran kaur': [100, 200, 500],
+          'manikaran chandok': [100, 200, 500, 680],
+          'manveen chandok': [10, 200, 65],
+          'sanjam patwalia': [200, 50]}
+
+
+# List of existing donor
+def existingdonors_list():
+    donor_list = "\n ".join(donor for donor in donors)
+    print("\nList of Existing Donors:", "\n", donor_list.title(), "\n")
+
+
+# Add donation to an existing donor
+def add_donation():
+        donor_selection = input("Enter 'Full Name' of the donor >")
+        donor_donation = input("\nEnter the 'Amount' donated >")
+        donor = donor_selection.strip().lower()
+        if donor in donors:
+            donor_amount = donors.get(donor)

```

Comment:
To append here, you don't need to create a new variable, you can append directly, like so:
donors[donor].append(donation_amount)